gsoc_contributors_guide.md

```diff
@@ -5,9 +5,74 @@ The Echoshader team aims to recruit talented [Google Summer of Code (GSoC)](http
+how netcdf defines common terms such as groups, dimensions, variables, and attributes.  
+
+* [Zarr](https://zarr.readthedocs.io/en/stable/) - a format for the storage of chunked, compressed, N-dimensional 
+arrays. Zarr has similar characteristics to netcdf, but has the added benefit of being able to be both read and wrote to
+concurrently from multiple threads or processes. Zarr provides a great overview of its [storage specifications](https://zarr.readthedocs.io/en/stable/spec/v2.html#hierarchies)
+that may be useful to read.    
+
+### Data Structures
+
+echopype was written to utilize as much functionality of [xarray](https://xarray.pydata.org/en/stable/index.html) 
+as possible. This is very beneficial as xarray is a library that can naturally work with our data, which is labelled 
+and multi-dimensional. A fantastic [xarray tutorial](https://xarray-contrib.github.io/xarray-tutorial/) has been put 
+together that describes the fundamentals of xarray. Be sure to become familiar with both DataArrays and Datasets as they
+are heavily used.     
+
+### Ocean Sonar Data
+
+For this project, you will be initially working with the output of the [compute_Sv](https://echopype.readthedocs.io/en/stable/api/echopype.calibrate.compute_Sv.html#echopype.calibrate.compute_Sv) 
+function. This is a function in echopype that computes the volume backscattering strength (Sv) of the provided EchoData
+object. This function returns an xarray Dataset that has several variables that are necessary for the visualization 
+of ocean sonar data. The Dataset has the dimensions and coordinates: 
+
+* frequency - transducer frequency used for the experiment, with units Hz
+
+* ping_time - timestamp of each ping
+
+* range_bin - sample index along a range
+
+The data variables of the Dataset are as follows, where items in parenthesis are the dimensions of the data variables: 
+
+* Sv (frequency, ping_time, range_bin) - volume backscattering strength measured from the echo
+
+* range (frequency, ping_time, range_bin) - the measured range of an echo in meters 
+
+* temperature - the temperature measurement of the water collected by the echosounder, with unit degree Celsius
+
+* salinity - the salinity measurement of the water collected by the echosounder, with unit part per thousand (PSU)
+
+* pressure - the pressure measurement of the water collected by the echosounder, with unit dbars
+
+* sound_speed (frequency, ping_time) - sound speed (in units m/s) for the provided temperature, salinity, and pressure 
+
+* sound_absorption (frequency, ping_time) - sea water absorption (in units dB/m) for each frequency and ping time, this 
+value is based on the temperature, salinity, pressure, and sound_speed
+
+* sa_correction (frequency) - the sa correction for each frequency
+
+* gain_correction - (frequency) - the gain correction for each frequency
+
+* equivalent_beam_angle (frequency) - the beam angle for each frequency
+
```

Comment:
I am not sure which data we plan to use. I believe that we still need to put together the data, see issue #3. I am currently using the newest stable version of Echopype and data from the Hake survey. I am not sure how this output changes with respect to the data. Maybe @leewujung can provide some insight here.

Reply:
@valentina-s: these variables were added in one of the v0.5.x releases so that the parameters that were used to calibrate the raw data into Sv are recorded. 

I think we should make sure all the datasets we share in this repo have the same structure/variables to avoid confusion. How many files are used in the echogram notebook you mentioned in #3 ?  It'll probably be a good idea that we re-convert them.

Reply:
@b-reyes : I think here it would help to use the inline backtick (like Sv --> `Sv`) so that contributors know that there are variables